src/org/battleship/Battlefield.java

```diff
@@ -0,0 +1,189 @@
+package org.battleship;
+
+import org.battleship.equipment.*;
+
+import java.util.*;
+
+public class Battlefield {
+    private final Cell[][] fieldOfBattle;
+
```

Comment:
Please keep list of ships also.
It will simplify a lot implementation of `checkIsAliveFleet`